resources/zib/zib-wounds.WoundCharacteristics.xml

```diff
@@ -0,0 +1,286 @@
+      <max value="1" />
+    </element>
+    <element id="Observation.component:woundLength.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="39126-8" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:woundLength.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="WoundLength" />
+      <definition value="The length of the decubitus wound applies similarly to the definitions of Wound Depth and Wound Width." />
+      <alias value="Wondlengte" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <mapping>
+        <identity value="zib-pressureulcer-v3.4-2020EN" />
+        <map value="NL-CM:19.1.9" />
+        <comment value="WoundLength" />
+      </mapping>
+      <mapping>
+        <identity value="zib-wound-v3.3-2020EN" />
+        <map value="NL-CM:19.2.12" />
+        <comment value="WoundLength" />
+      </mapping>
+    </element>
+    <element id="Observation.component:woundWidth">
+      <path value="Observation.component" />
+      <sliceName value="woundWidth" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:woundWidth.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="39125-0" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:woundWidth.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="WoundWidth" />
+      <definition value="The width of the decubitus wound." />
+      <alias value="Wondbreedte" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <mapping>
+        <identity value="zib-pressureulcer-v3.4-2020EN" />
+        <map value="NL-CM:19.1.10" />
+        <comment value="WoundWidth" />
+      </mapping>
+      <mapping>
+        <identity value="zib-wound-v3.3-2020EN" />
+        <map value="NL-CM:19.2.13" />
+        <comment value="WoundWidth" />
+      </mapping>
+    </element>
+    <element id="Observation.component:woundDepth">
+      <path value="Observation.component" />
+      <sliceName value="woundDepth" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:woundDepth.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="39127-6" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:woundDepth.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="WoundDepth" />
+      <definition value="The depth of the decubitus wound or pressureulcer." />

```

Comment:
Change this to 'The depth of the (decubitus) wound.'.

Reply:
fix